packages/rooks/src/hooks/useAsyncEffect.ts

```diff
@@ -48,8 +47,9 @@ function useAsyncEffect<T>(
     } catch (error) {
       throw error;
     }
+
     // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, [effect, getIsMounted, deps]);
```

Comment:
This was causing continuous the callback to update too many times for no reason if the dependency was passed as `useAsynEffect(callback, [])` because the array is recreated on each render.

Reply:
Interesting. Nice fix!